cas-server-core/src/main/java/org/jasig/cas/services/DefaultRegisteredServiceAccessStrategy.java

```diff
@@ -60,6 +62,12 @@
     /** Is the service allowed to use SSO? **/
     private boolean ssoEnabled = true;
 
+    private String startingDateTime;
```

Comment:
Yes, that's what I meant: a new `TimeBasedRegisteredServiceAccessStrategy` extending the default one...